precompiles/avs/avs.go

```diff
@@ -165,7 +165,7 @@ func (p Precompile) Run(evm *vm.EVM, contract *vm.Contract, readOnly bool) (bz [
 		}
 	// queries
 	case MethodGetOptinOperators:
-		bz, err = p.GetOptedInOperatorAccAddrs(ctx, contract, method, args)
+		bz, err = p.GetOptedInOperatorAccAddress(ctx, contract, method, args)
```

Comment:
Plural, since it returns multiple of these
```suggestion
		bz, err = p.GetOptedInOperatorAccAddresses(ctx, contract, method, args)
```

Reply:
fixed